docs/guide/extending_ez_platform_ui.md

```diff
@@ -315,294 +315,245 @@ In this example the plugin is added as a dependency of the Dashboard block view,
 
 After this configuration is complete the Dashboard should display the new block.
 
-## Extending the Landing Page (Enterprise)
+!!! enterprise
 
-### Extending the Form Builder
+    ## Extending the Form Builder
 
-EZ ENTERPRISE
+    Form Builder Bundle comes a number of types of fields, but it was designed to be easy to extend by adding new ones.
 
-Form Builder Bundle currently comes with three basic types of fields: Single Line Text, Paragraph Text and Email.
+    ### Field definitions
 
-It was designed to be easy to extend by adding new types of fields.
+    Default field definitions are available in `Resources\config\default_fields.yml`.
 
-#### Field definitions
+    #### Field definition structure
 
-Default field definitions are available in `Resources\config\default_fields.yml`.
+    Field definitions are contained under the `fields` key. Each definition has its own key, e.g. `single_line_text`. Each definition must contain two sections:
 
-##### Field definition structure
+    - `identifier` - name of the definition used on the front end
+    - `displayName` - name displayed in the Page mode editor in the `fields` tab
 
-Field definitions are contained under the `fields` key. Each definition has its own key, e.g. `single_line_text`. Each definition must contain two sections:
+    The definition can also contain the following optional sections:
 
-- `identifier` - name of the definition used on the front end
-- `displayName` - name displayed in the Page mode editor in the `fields` tab
+    - `validators` - defines validators that the field will use. This must contain the validator's identifier and the values that will be checked during validation, for example:
```

Comment:
But these are not "Fields" in the meaning of content Fields, but form fields, which are a different term altogether. I use small letter here on purpose so that they don't get confused with Field as a domain term

Reply:
In *this* case it's the Field!